src/components/reference-editor/spec.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import test from 'tape';
+import { shallow, mount } from 'enzyme';
+import spy, { createSpy } from '../../utils/spy';
+
+import {
```

Comment:
Yeah... I feel like we could benefit from an eslint rule to detect unused identifiers